prov/sockets/src/sock_conn.c

```diff
@@ -579,8 +582,9 @@ int sock_ep_connect(struct sock_ep_attr *ep_attr, fi_addr_t index,
 		conn_fd = -1;
 	}
 
-	SOCK_LOG_ERROR("Connect error, retrying - %s - %d\n",
-		       strerror(ofi_sockerr()), conn_fd);
+	ofi_straddr(buf, &buf_len, FI_SOCKADDR, &addr.sa);
```

Comment:
Use ofi_straddr_log() instead, so we only format the address if we're going to log it as an error.

Reply:
done